test/utils/math/SignedMath.t.sol

```diff
@@ -48,7 +48,8 @@ contract SignedMathTest is Test {
     }
 
     // 2. more complex test, full int256 range
-    function testAverage2(int256 a, int256 b) public pure {
+    /// @custom:halmos --solver-timeout-assertion 0
```

Comment:
Based on [this](https://github.com/a16z/halmos/blob/ba94bdc232c746b61f12eeacef6136eace81c7c2/tests/regression/halmos.toml#L154), I understand that this is just removing the timeout.

Since it's not well documented I would leave a note:


```suggestion
    /// (timeout 0 means no timeout)
    /// @custom:halmos --solver-timeout-assertion 0
    function testSymbolicAverage2(int256 a, int256 b) public pure {
```